tasks/ATOMIC - Required Objects 5/batch.csv

```diff
@@ -1,663 +1,101 @@
-Determine objects needed to complete events.,"Given a short event description, determine what objects are needed to complete the event.","annotation, data collection, event, objects",PersonX throws acid in the eyes,PersonX pushes PersonY's cabinet far,PersonX raises PersonX's hands to PersonX's lips,{},PersonX throws acid in the eyes|PersonX pushes PersonY|PersonX raises PersonX,hold acid,{},{},{},glove,{},{},{},{},{},{},{},{},{},{},{},{},{},{},{},{},{},{},{},1.0,1.0,0.0,1.0,0.0,1.0
-Determine objects needed to complete events.,"Given a short event description, determine what objects are needed to complete the event.","annotation, data collection, event, objects",PersonX receives another pizza,PersonX goes fishing salmon,PersonX gives people letters,{},PersonX receives another pizza|PersonX goes fishing salmon|PersonX gives people letters,{},{},{},{},{},{},{},{},{},{},{},{},{},{},{},{},{},{},{},{},{},{},{},{},0.0,1.0,0.0,1.0,,0.0
-Determine objects needed to complete events.,"Given a short event description, determine what objects are needed to complete the event.","annotation, data collection, event, objects",PersonX gets check from the bank,PersonX quickly left payment,PersonX swings club hard,{},PersonX gets check from the bank|PersonX quickly left payment|PersonX swings club hard,{},{},{},{},{},{},{},{},{},{},{},{},{},{},{},{},{},{},{},{},{},{},{},{},0.0,1.0,,0.0,,0.0
-Determine objects needed to complete events.,"Given a short event description, determine what objects are needed to complete the event.","annotation, data collection, event, objects",PersonX eventually got the dog,PersonX gives a kitten away,PersonX bestows upon PersonY the throne,{},PersonX eventually got the dog|PersonX gives a kitten away|PersonX bestows upon PersonY the throne,{},{},{},{},{},{},{},{},{},{},{},{},{},{},{},{},{},{},{},{},{},{},{},{},0.0,1.0,0.0,1.0,0.0,1.0
-Determine objects needed to complete events.,"Given a short event description, determine what objects are needed to complete the event.","annotation, data collection, event, objects",PersonX wears dresses anymore,PersonX leaves the car quickly,PersonX places a block in the way,{},PersonX wears dresses anymore|PersonX leaves the car quickly|PersonX places a block in the way,{},{},{},{},{},{},{},{},{},{},{},{},{},{},{},{},block people from entering,transport the block,{},{},concrete cube,car,{},{},,0.0,0.0,1.0,1.0,1.0
-Determine objects needed to complete events.,"Given a short event description, determine what objects are needed to complete the event.","annotation, data collection, event, objects",PersonX glues picture back together,PersonX holds the toy in PersonX's hands,PersonX enters hotdog eating contest,{},PersonX glues picture back together|PersonX holds the toy in PersonX|PersonX enters hotdog eating contest,fix the photo,apply adhesive to,{},{},adhesive,print,{},{},carry around,grasp in fingers,{},{},doll,a plaything,{},{},wash down the food,throw up in,{},{},drink,trash can,{},{},1.0,1.0,
```

Comment:
Removed the `Answer.feedback`.
The reason for those brackets are because those fields are optional or conditioned on other fields. I think we should treat them as blank?